Yoast/wordpress-seo
# Changed section header to match old snippet preview

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:


* Install deps: `composer install & yarn install`.
* Link the yoast-components branch:
  * In the yoast-components clone: `yarn link` (skip this step if already done before).
  * In wordpress-seo clone: `yarn link yoast-components`.
* Checkout it's companion branch in yoast-components. https://github.com/Yoast/yoast-components/pull/491
* Install and build yoast-components: `yarn install && yarn start`.
* Start the wordpress-seo dev server: `yarn start`.
* In the `wp-config.php` file of your WordPress installation add `define( 'YOAST_FEATURE_SNIPPET_PREVIEW', true );` to enable the React SnippetPreview.
* Check if the new snippet preview is the same as the old snippet preview. Addressed for now is the following:
* Check if the new snippet preview is the same as the old snippet preview. Addressed in this pull is  the following:
```
Change section header to 16px.
```
## Quality assurance


Fixes #9533
